core/src/main/scala/cats/evidence/Is.scala

```diff
@@ -65,7 +66,19 @@ abstract class Is[A, B] extends Serializable {
     substitute[A =:= ?](implicitly[A =:= A])
 }
 
-object Is {
+sealed abstract class IsInstances {
+  import Is._
+
+  /**
+   * The category instance on Leibniz categories.
+   */
+  implicit val leibniz: Category[Is] = new Category[Is] {
```

Comment:
Can we give thise one a more verbose name? Something like `catsEvidenceCategoryForLeibniz`? :)

Reply:
The `Category` instance for `As` is named as `liskov` -- so I copied that :-)

Reply:
Good call :)